tests/gtests/dnnl_test_common.hpp

```diff
@@ -1093,7 +1094,7 @@ template <typename pd_t, typename hint_pd_t, typename... prim_params_t>
 void test_bwd_pd_constructors(const pd_t &pd, const hint_pd_t &hint,
         const allows_attr_t &aa, const prim_params_t &...prim_params) {
     auto test_pd = pd_t();
-    auto hint_pd = hint;
+    const auto &hint_pd = hint;
```

Comment:
nit: you can just rename the parameter
```suggestion
void test_bwd_pd_constructors(const pd_t &pd, const hint_pd_t &hint_pd,
        const allows_attr_t &aa, const prim_params_t &...prim_params) {
    auto test_pd = pd_t();
```

Reply:
This bright idea didn't come to me. Thanks, updated.